### PR TITLE
 Added IBAN to payee mappings

### PR DESCRIPTION
**Pull request by PehrGit**
Hi,

I recently opened #27 and today I thought to myself: "I'm a programmer, why not do it myself".

So here it is, I added the counterparty IBAN to the payee field for Knab/Rabo/ING and adjusted the concatenation to include a space between values and trim any excess whitespace. My compliments for the way you set this up, it was very easy to make these adjustments.

Let me know what you think and if you'd like to see anything changed. 👍

### CODE REVIEW

**Comment by DanielsWrath** [comment]
I will give it a spin this weekend! Will keep you up to date when I merge this 😄 
Thanks for the effort!

File: banks.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     ],
     "dateFormat": "DD-MM-YYYY",
     "payee": [
+      "Tegenrekeningnummer",
       "Tegenrekeninghouder"
```
Comment (DanielsWrath) [inline banks.json:32]:
```suggestion
      "Tegenrekeninghouder",
      "Tegenrekeningnummer"
```

File: banks.json [hunk 2]
```diff
@@ -82,6 +83,7 @@
     ],
     "dateFormat": "YYYY-MM-DD",
     "payee": [
+      "Tegenrekening IBAN/BBAN",
       "Naam tegenpartij"
```
Comment (DanielsWrath) [inline banks.json:87]:
```suggestion
      "Naam tegenpartij",
      "Tegenrekening IBAN/BBAN"
```

File: banks.json [hunk 3]
```diff
@@ -121,6 +123,7 @@
     ],
     "dateFormat": "YYYYMMDD",
     "payee": [
+      "Tegenrekening",
       "Naam / Omschrijving"
```
Comment (DanielsWrath) [inline banks.json:127]:
```suggestion
      "Naam / Omschrijving",
      "Tegenrekening"
```

File: js/convert.js [hunk 4]
```diff
@@ -176,8 +176,14 @@ const BankMapper = function (bank) {
     const getLine = (line, fieldList) => {
         let returnLine = "";
 
-        for (const field of fieldList)
-            returnLine += line[field];
+        for (const field of fieldList){
+            var fieldValue = line[field].trim();
+            if(!!returnLine.trim() && !!fieldValue){
```
Comment (DanielsWrath) [inline js/convert.js:181]:
```suggestion
            if (!!returnLine.trim() && !!fieldValue) {
```

File: js/convert.js [hunk 5]
```diff
@@ -176,8 +176,14 @@ const BankMapper = function (bank) {
     const getLine = (line, fieldList) => {
         let returnLine = "";
 
-        for (const field of fieldList)
-            returnLine += line[field];
+        for (const field of fieldList){
+            var fieldValue = line[field].trim();
```
Comment (DanielsWrath) [inline js/convert.js:180]:
```suggestion
            const fieldValue = line[field].trim();
```

File: js/convert.js [hunk 6]
```diff
@@ -176,8 +176,14 @@ const BankMapper = function (bank) {
     const getLine = (line, fieldList) => {
         let returnLine = "";
 
-        for (const field of fieldList)
-            returnLine += line[field];
+        for (const field of fieldList){
```
Comment (DanielsWrath) [inline js/convert.js:179]:
```suggestion
        for (const field of fieldList) {
```

File: js/convert.js [hunk 7]
```diff
@@ -176,8 +176,14 @@ const BankMapper = function (bank) {
...
+            var fieldValue = line[field].trim();
+            if(!!returnLine.trim() && !!fieldValue){
+                returnLine += " ";
+            }
```
Comment (DanielsWrath) [inline js/convert.js:183]:
```suggestion
            if (!!returnLine.trim() && !!fieldValue)
                returnLine += " ";
```

**Comment by PehrGit** [comment]
Hey, sorry for the late reply. It looks like you took care of the revisions yourself, thanks!

**Comment by DanielsWrath** [comment]
No problem! Hope it works for you now

**Pull request merged**